go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/cilium/ebpf v0.10.0 // indirect
 	github.com/coreos/go-systemd/v22 v22.5.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```

Comment:
why bumping this in the PR. I think leaving renovate will autogenerate the changelog for us WDYT?

Reply:
sadly since it is an indirect dependency is not taken into account by renovate. 
I think @sigilioso was looking into it

Reply:
actually, this might be required right now 
https://github.com/newrelic/infrastructure-bundle/actions/runs/5142180991/jobs/9255589710#step:10:50

Reply:
oh and just noticed that it was failing in previous commit of this branch